WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -98,6 +98,10 @@ class GutenbergViewController: UIViewController, PostEditor {
     }
 
     func setHTML(_ html: String) {
+        guard gutenberg.isLoaded else {
+            return
+        }
```

Comment:
At the beginning I couldn't be quite sure about choosing a guard or just an if around gutenberg.updateHtml(html)

I tested different cases and it looks like it is working OK this way so let's go with it 🎉

Reply:
Than you!
In general, if it's an early exit, I prefer to use guard, even if the rest of the function is short.
But an if statement should work fine as well 👍

Reply:
I prefer guard for an early exit also, I was thinking something like below instead. but as I said before it works OK in its current version also 👍 

```
    func setHTML(_ html: String) {
        self.html = html
        if gutenberg.isLoaded {
            gutenberg.updateHtml(html)
        }
    }
```